Slim/Schema.pm

```diff
@@ -3293,6 +3303,17 @@ sub canFulltextSearch {
 	return $canFulltextSearch;
 }
 
+sub _workRequired {
+	if ( $scanWorks == SCAN_WORKS_FOR_MY_CLASSICAL_GENRES ) {
```

Comment:
Following the above comment this should be changed to something like:

```perl
If ((defined $scanWorks : $scanWorks : $prefs->get('worksScan')) == SCAN_WORKS_FOR_MY_CLASSICAL_GENRES) {
...
```

Reply:
BTW: nice shortcut to not have three `return` statements as before.

Reply:
Mostly enabled by getting the option values the right way around as you suggested :)